Signal/src/ViewControllers/ThreadSettings/OWSConversationSettingsViewController.m

```diff
@@ -289,8 +289,7 @@ - (void)updateTableContents
                              itemWithCustomCellBlock:^{
                                  NSString *colorName = self.thread.conversationColorName;
                                  UIColor *currentColor =
-                                     [UIColor ows_conversationColorForColorName:colorName
-                                                                       isShaded:Theme.isDarkThemeEnabled];
+                                     [UIColor ows_conversationColorsForColorName:colorName].themeColor;
```

Comment:
This is a nullable method - should this be ows_conversationColorsOrDefaultForColorName?